dll/win32/shell32/shell32.spec

```diff
@@ -178,8 +178,8 @@
 179 stdcall SHGetNewLinkInfoA(str str ptr long long)
 180 stdcall SHGetNewLinkInfoW(wstr wstr ptr long long)
 181 stdcall -noname RegisterShellHook(long long)
-182 varargs ShellMessageBoxW(long long wstr wstr long)
-183 varargs ShellMessageBoxA(long long str str long)
+182 varargs ShellMessageBoxW(ptr ptr wstr wstr long)
```

Comment:
I think there will be a conflict here wrt. master.

Reply:
Yeah, rebased.